sycl/gdb/libsycl.so-gdb.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, class_type, result_type, depth):
         self.depth = depth
 
     def get_arg_types(self):
-        return gdb.lookup_type("cl::sycl::id<%s>" % self.depth)
+        return gdb.lookup_type("sycl::_V1::id<%s>" % self.depth)
```

Comment:
Not sure how this file is being tested, so the changes here are "blind".

Reply:
@stevemerr, do you know?

Reply:
I suspect any end-to-end testing that would run gdb-oneapi on a sycl sample would exercise this file.

Reply:
Does this need to be `sycl::_V1::id` or can we just do `sycl::id`?